resources.html

```diff
@@ -159,6 +160,51 @@ <h5 class="card-title">Tutorial for Application Developers</h5>
         </div>
     </section>
     
+    <hr class="hairline">
+
+    <!-- Integration section -->
+    <section class="odd-section">
+        <div class="container" style="padding-bottom: 48px;">
+            <div class="row">
+                <div class="col-lg-12 text-center">
+                    <h3 class="section-heading" style="padding-top: 48px;">Integrate with Other Products</h3>
+                    <p>Integrating LoopBack with other products and libraries.</p>
+                    <p style="height:24px"></p>
+                </div>
+            </div>
+
+            <div class="row text-center" style="justify-content: center">
+                <div class="col-sm-12 col-md-6">
+                    <a class="card-2" href="openapi-to-graphql.html" target="_blank" rel="noopener noreferrer">
+                        <div class="row">
+                            <div class="col-sm-3">
+                                <img class="icon" src="images/homepage/conveyorbelt-2x.png" alt="OpenAPI-to-GraphQL Icon"/>
+                            </div>
+                            <div class="col-sm-9">
+                                <h5 class="card-title">openapi-to-graphql</h5>
```

Comment:
very small nitpick but can we keep the capitalization the same as the page's: `OpenAPI-to-GraphQL`

Reply:
I was thinking to keep the module name. But can change it to `OpenAPI-to-GraphQL`.